waspc/src/Wasp/Generator/Job.hs

```diff
@@ -33,8 +29,3 @@ data JobMessageData
 data JobOutputType = Stdout | Stderr deriving (Show, Eq)
 
 data JobType = WebApp | Server | Db | Wasp deriving (Show, Eq, Ord, Bounded, Enum)
-
--- | We are importing relevant types per executor e.g. JobFn, this functions maps
---   the executor to the import path of the relevant types.
-jobExecutorTypesImportPathFromSdk :: JobExecutor -> Path Posix (Rel r) File'
-jobExecutorTypesImportPathFromSdk PgBoss = makeSdkImportPath [relfileP|jobs/pgBoss/types|]
```

Comment:
It's unusual for this file to import stuff from specific generators. It should be at the bottom of the generator's import dependency graph (everything in the generator imports this guy, but it doesn't import anything from the generator).

This is important because Haskell doesn't tolerate indicrect circular imports. And they're a pain when they happen.